### ISSUE TITLE
The screen reader is not announcing the status message of which file is selected after selecting the select subscription button. 

### ISSUE THREAD

**Issue by prasadaps** [microsoft/vscode#127079]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yWCAG;#DesktopApp;#Visual Studio Code Client;#JAWS;#NVDA;#WCAG4.1.3;#DesktopApp;#Win10;#VisualStudioCode-Win32-Apr2021;

### Environment details:
Application Name: Azure Stream Analytics Tools for Visual Studio Code
Windows Version: Win10 
Visual studio code  version : -1.55.1(User Setup)
Screenreader NVDA(2020.4), JAWS(2021.2102.34ILM)

### Repro Steps:

1. Launch Visual Studio code application with installed Azure Stream Analytics Tools extension . 
2. Tab to "Explorer" section in activity bar. 
3. Press control +shift +P and edit bar will appear. 
4. Type "ASA:Create new project" and press enter to create project. 
5. New project will get created. 
6. Tab to Input item in Project Tree Section 
7. Right click on input and navigate to "ASA : Add Input" using arrow key and press enter. 
8. Select IoT hub from available option using enter key. 
9. Select "select from azure subscriptions" from available option using enter key. 
10. Type any name for input file and press enter to create Input json file. 
11. Tab to "Select Subscription" button and press enter.
12. Start screenreader and select "AML V1 Personal 2" and observe that after selecting this file screen reader does not announce any information.

### Actual Result:
After selecting the select "AML V1 Personal 2" file  the screen reader does not announce any information that the AML V1 Personal 2 file is selected for subscription.

This issue is observed with both screen  reader NVDA and JAWS.

### Expected Result:
The screen after selecting the selecting the "AML V1 Personal 2" file the screen reader should announce that this file is selected for subscription.

### User Impact:
As ScreenReader is not announcing the status message the screen reader users may not get the acknowledge that the select "AML V1 Personal 2" file is selected as subscription.

### Attachment:
[A11y_AzureStreamAnalyticsToolsForVisualStudioCode_Input_SelectSubscription.zip](https://github.com/microsoft/vscode/files/6709054/A11y_AzureStreamAnalyticsToolsForVisualStudioCode_Input_SelectSubscription.zip)

**Comment by isidorn** [comment]
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. Thanks!
